tekton/pipelines/cncf.yaml

```
@@ -28,6 +28,9 @@ spec:
     runAfter: [create-release]
     taskRef:
       name: create-cluster
+    params:
+    - name: release-id
+      value: "$(tasks.get-latest-release.results.release-id)"
```

In this pipeline, we create a release _on the fly_, so we should use the result of that task which is called `create-release`.

```suggestion
      value: "$(tasks.create-release.results.release-id)"
```

---

Oops, I copied and pasted without looking closely enough. This is the value I used in my manual test.